packages/api-client/src/views/Request/RequestSidebarItem.vue

```diff
@@ -150,7 +150,12 @@ const showChildren = computed(
           <div class="relative">
             <RequestSidebarItemMenu
               v-if="!isReadOnly"
-              :item="item" />
+              :item="item"
+              :parentUid="
```

Comment:
Does typescript not like this?

```suggestion
              :parentUid="parentUids[parentUids.length - 1]"
```